tests/platform_tests/cli/test_show_platform.py

```diff
@@ -89,10 +89,12 @@ def test_show_platform_summary(duthosts, enum_rand_one_per_hwsku_hostname, dut_v
 
 
 def test_show_platform_syseeprom(duthosts, enum_rand_one_per_hwsku_hostname, dut_vars):
+
```

Comment:
@vdahiya12 can we remove this empty line?

Reply:
fixed